homeassistant/components/binary_sensor/abode.py

```diff
@@ -23,59 +23,59 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up a sensor for an Abode device."""
-    data = hass.data.get(DATA_ABODE)
+    import abodepy.helpers.constants as CONST
+
+    device_types = [
```

Comment:
It has to be defined in a function because it's constants imported from a third-party lib.